articles/aks/keda-integrations.md

```diff
@@ -52,12 +52,9 @@ However, these external scalers aren't supported as part of the add-on and rely
 
 <!-- LINKS - internal -->
 [aks-support-policy]: support-policies.md
-[azure-monitor]: ../azure-monitor/overview.md
-[azure-monitor-container-insights]: ../azure-monitor/containers/container-insights-onboard.md
 [keda-arm]: keda-deploy-add-on-arm.md
 
 <!-- LINKS - external -->
-[keda-scalers]: https://keda.sh/docs/scalers/
-[keda-metrics]: https://keda.sh/docs/latest/operate/prometheus/
-[keda-event-docs]: https://keda.sh/docs/2.7/operate/events/
```

Comment:
Looks like https://github.com/MicrosoftDocs/azure-docs-pr/pull/200520 was merged without handling the comment; thanks for aligning it!

Reply:
Yes, I thought of creating this PR after seeing that PR is merged.